typescript-getting-started/functions/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "lib": ["es6", "es2015.promise"],
+    "module": "commonjs",
+    "noImplicitAny": false,
+    "outDir": "",
+    "sourceMap": true,
+    "target": "es5",
```

Comment:
target es6 to improve the readability and presumedly speed of the transpiled source.

Reply:
good idea